studio/schemas/objects/sections/employeeHighlight.ts

```diff
@@ -0,0 +1,56 @@
+import { defineField } from "sanity";
+
+import { isInternationalizedString } from "studio/lib/interfaces/global";
+import { titleID } from "studio/schemas/fields/text";
+import { firstTranslation } from "studio/utils/i18n";
+
+export const employeeHighlightID = "employeeHighlight";
+
+export const employeeHighlightSection = defineField({
+  name: employeeHighlightID,
+  title: "Employee Highlight",
+  type: "object",
+  fields: [
+    {
+      name: titleID.basic,
+      type: "internationalizedArrayString",
+      title: "Title",
+      description: "The title/prefix that will appear above the name block.",
+    },
+    {
+      name: "name",
+      type: "string",
+      title: "Name",
+      description: "The name of the employee.",
+    },
+    {
+      name: "description",
+      type: "internationalizedArrayString",
```

Comment:
Perhaps we should use internationalizedArrayText for the description, so the text is in a block rather than one line in Sanity 🤓

Reply:
Fixed in https://github.com/varianter/variant.no/pull/943/commits/dde357498e6bd27d79da33b6545ac66aa5841877